2FA article for Umbraco 15 (#6821)
umbraco/UmbracoDocs: 15/umbraco-cms/reference/security/two-factor-authentication.md

    }
}
```

{% endcode %}

At this point, the 2FA is active, but no members have set up 2FA yet. The setup of 2FA depends on the type. In the case of App Authenticator, add the following to the **view** showing the edit profile of the member.

> 7. Add or choose a members-only page that should have the two-factor authentication setup.

Author: I've updated the paragraph, and added a hint about how you can skip a step if you already have a page set up of editing member profiles.

> 7. Add or choose a members-only page that should have the two-factor authentication setup.

Author: @eshanrnh - let me know if it makes sense.

> 7. Add or choose a members-only page that should have the two-factor authentication setup.

Reviewer: Looks good to me.

 }
}
```

{% endcode %}

8. Update the `namespace` on line 4 to match your project.

> With the 2FA in place, the provider needs to be registered in the backoffice client in order for the user to be able to use it.

Reviewer: ```suggestion
With the 2FA in place, the provider needs to be registered in the backoffice client so the user can use it.
```

Reviewer: suggestion to shorten the sentence.
